src/Bicep.Core/Registry/Providers/TypesV1Archive.cs

```diff
@@ -44,4 +52,14 @@ private static async Task AddFileToTar(TarWriter tarWriter, string archivePath,
 
         await tarWriter.WriteEntryAsync(tarEntry);
     }
+
+    private static IEnumerable<string> getAllUniqueTypePaths(string pathToIndex, IFileSystem fileSystem)
```

Comment:
[nit] The convention we follow is Pascal Case (rather than camel case) for method names.
```suggestion
    private static IEnumerable<string> GetAllUniqueTypePaths(string pathToIndex, IFileSystem fileSystem)
```